mteb/models/e5_instruct.py

```diff
@@ -66,6 +66,15 @@ def encode(
     open_source=True,
     revision="baa7be480a7de1539afce709c8f13f833a510e0a",
     release_date=E5_PAPER_RELEASE_DATE,
+    framework=["GritLM", "PyTorch"],
+    similarity_fn_name="cosine",
+    use_instuctions=True,
+    reference="https://huggingface.co/intfloat/multilingual-e5-large-instruct",
+    n_parameters=560_000_000,
+    memory_usage=None,
+    embed_dim=1024,
+    license="mit",
+    max_tokens=514,
```

Comment:
Is this true? 514 seems like an odd number.

Reply:
Yes, this is true. I think that this is 512 + 2 (query and passage)